### ISSUE TITLE
Implement reading of command line arguments

### ISSUE THREAD

**Issue by zerny** [dart-lang/fletch#18]
There is currently no support for reading arguments passed as parameters to Dart\Fletch programs/snapshots.

**Comment by mit-mit** [comment]
@skabet we tagged this for FixIt, any idea how complicated this is?

**Comment by skabet** [comment]
Yep, I'd say medium complexity. First CL is up here: https://codereview.chromium.org/1526473002/. A follow-up would target the `fletch` command line utility.
